packages/turf-mask/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Takes a {@link Polygon|polygon} and an optional mask and returns the outside area,
+ * if you don't pass a mask polygon then the world is used.
+ *
+ * @name mask
+ * @param {Feature<Polygon>} polygon GeoJSON Polygon as input
```

Comment:
Mask should also include as input: `FeatureCollection<Polygon>` & `Feature<MultiPolygon>`.

**Input two polygons**
https://gist.github.com/69de7993a2e8f485599f28b74a75b6a2
![image](https://cloud.githubusercontent.com/assets/550895/23536068/934840de-ff90-11e6-97f9-28fd282ba20c.png)

**Returns Mask with two holes**
https://gist.github.com/f44af2532298fd531ea13e27f0cf264c
![image](https://cloud.githubusercontent.com/assets/550895/23536122/de254d54-ff90-11e6-80fb-fa446f08be08.png)